File: app/src/main/java/com/Fachhochschulebib/fhb/pruefungsplaner/data/Uuid.java
package com.Fachhochschulebib.fhb.pruefungsplaner.data;


import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "Uuid")
public class Uuid {

    @PrimaryKey()
    @NonNull
    @ColumnInfo(name = "uuid")
    private String Uuid;

    public String getUuid() {
        return Uuid;
    }

    public void setUuid(String uuid) {
        Uuid = uuid;
    }
}

File: app/src/main/java/com/Fachhochschulebib/fhb/pruefungsplaner/data/Uuid.kt
package com.Fachhochschulebib.fhb.pruefungsplaner.data

import androidx.annotation.NonNull
import androidx.room.PrimaryKey
import androidx.room.ColumnInfo
import androidx.room.Entity

@Entity(tableName = "Uuid")
class Uuid {
    @PrimaryKey
    @ColumnInfo(name = "uuid")
    @NonNull
    var uuid: String? = null
}